portal-web/test/functional/com/liferay/portalweb/macros/MultiFactorAuthentication.macro

```diff
@@ -123,6 +123,31 @@ definition {
 		}
 	}
 
+	macro configureOTPwithOrderUpdate {
```

Comment:
This looks very similar to the `configureOTP` macro. Can't we just logic to specify something in the order field?

Reply:
I extend the configureOTP macro.